bin/acceptance-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+
+case $1 in
+    chrome)
+        BROWSER_NAME=chrome
+        PLATFORM="Windows 10"
```

Comment:
Add in `BROWSER_VERSION=latest`?

Reply:
Added in https://github.com/mozmeao/www-config/pull/243/commits/3b8265b42dbf35703165f1875f798069a9987f23